### PR TITLE
Add ansible remediation for UBTU-20-010075

### PR DESCRIPTION
**Pull request by dexterle**
#### Description:

- Fix UBTU-20-010075
- Add Ansible remediation for `accounts_passwords_pam_faildelay_delay`

#### Rationale:

- Part of Ubuntu 20.04 DISA STIG v1r9 profile upgrade
- https://github.com/ComplianceAsCode/content/pull/10738

#### Review Hints:

Build the product:
```
./build_product ubuntu2004
```
To test these changes with Ansible:
```
ansible-playbook build/ansible/ubuntu2004-playbook-stig.yml --tags "DISA-STIG-UBTU-20-010075"
```

Checkout Manual STIG OVAL definitions, and use software like DISA STIG Viewer to view definitions.
```
git checkout dexterle:add-manual-stig-ubtu-20-v1r9
```

This STIG can be tested with the latest Ubuntu 2004 Benchmark SCAP. For reference, please review the latest artifacts: https://public.cyber.mil/stigs/downloads/

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faildelay_delay/ansible/ubuntu.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_ubuntu
```
Comment (marcusburghardt) [inline linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faildelay_delay/ansible/ubuntu.yml:1]: Please, rename this file to `shared.yml`. This remediation might be useful for other products too.

**Review by teacup-on-rockingchair — APPROVED** [review]
Nice improvement :bow:  have nothing to add besides @marcusburghardt note

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faildelay_delay/ansible/ubuntu.yml (hunk 1 above)
Comment (dodys) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faildelay_delay/ansible/ubuntu.yml:1]: please also consolidate the bash remediation into a single `shared.sh`

**Review by dodys — CHANGES_REQUESTED** [review]

**Comment by dodys** [comment]
@dexterle I wonder if we can change this PR to instead use the bash and ansible remediation from the template. Those were added after both Ubuntu's and Suse's remediation already existed. It might be the time to consolidate everything into the template.
@teacup-on-rockingchair if you could also test using the template remediation on your side ^
I will share my results later

**Comment by dodys** [comment]
> @dexterle I wonder if we can change this PR to instead use the bash and ansible remediation from the template. Those were added after both Ubuntu's and Suse's remediation already existed. It might be the time to consolidate everything into the template. @teacup-on-rockingchair if you could also test using the template remediation on your side ^ I will share my results later

The template seems enough and the tests are still passing

**Comment by teacup-on-rockingchair** [comment]
> > @dexterle I wonder if we can change this PR to instead use the bash and ansible remediation from the template. Those were added after both Ubuntu's and Suse's remediation already existed. It might be the time to consolidate everything into the template. @teacup-on-rockingchair if you could also test using the template remediation on your side ^ I will share my results later
> 
> The template seems enough and the tests are still passing



> @dexterle I wonder if we can change this PR to instead use the bash and ansible remediation from the template. Those were added after both Ubuntu's and Suse's remediation already existed. It might be the time to consolidate everything into the template. @teacup-on-rockingchair if you could also test using the template remediation on your side ^ I will share my results later

Works ok on my setup for SLES :+1:

**Comment by jan-cerny** [comment]
ping

**Comment by dodys** [comment]
dexterle reached out last week about continuing work on this PRs, and I've asked to open new ones for a clean state. Therefore closing this one.

**Pull request closed without merging**